Jenkinsfile-Internal-Release

```diff
@@ -52,32 +60,32 @@ node('ubuntu-zion') {
       if (params.nexus_repository_manager_version) {
         stage('Update Repository Manager Version') {
           OsTools.runSafe(this, "git checkout ${branch}")
-          def javaVersionsDockerfilesMap = [
-              (JAVA_8): DOCKERFILE_JAVA_8,
-              (JAVA_11): DOCKERFILE_JAVA_11,
-              (JAVA_17): DOCKERFILE_JAVA_17
-          ]
-          javaVersionsDockerfilesMap.each { javaVersion, dockerfile ->
-            updateRepositoryManagerVersion("${pwd()}/${dockerfile}", javaVersion)
+          dockerfileMap.each { javaVersion, dockerfiles ->
+            dockerfiles.each { dockerfile ->
+              updateRepositoryManagerVersion("${pwd()}/${dockerfile}", javaVersion)
+            }
           }
           version = getShortVersion(params.nexus_repository_manager_version)
         }
       }
     }
-    stage('Build') {
-      def dockerfilesMap = [
-          (OPENJDK8): DOCKERFILE_JAVA_8,
-          (OPENJDK11): DOCKERFILE_JAVA_11,
-          (OPENJDK17): DOCKERFILE_JAVA_17
-      ]
-      def dockerfilePath = dockerfilesMap.get(params.java_version)
+    def dockerfilePath = dockerfileMap[params.java_version][0]
+    def alpineDockerfilePath = params.java_version == OPENJDK8 ? null : dockerfileMap[params.java_version][1]
+
+    stage('Build UBI Image') {
       def baseImage = extractBaseImage(dockerfilePath)
       def baseImageRefFactory = load 'scripts/BaseImageReference.groovy'
       def baseImageReference = baseImageRefFactory.build(this, baseImage as String)
       def baseImageReferenceStr = baseImageReference.getReference()
       def hash = OsTools.runSafe(this, "docker build --quiet --label base-image-ref='${baseImageReferenceStr}' --no-cache --tag ${imageName} . -f ${dockerfilePath}")
       imageId = hash.split(':')[1]
     }
+    if (params.java_version != OPENJDK8) {
+      stage('Build Alpine Image') {
+        def hash = OsTools.runSafe(this, "docker build --quiet --no-cache --tag ${imageName}-alpine . -f ${alpineDockerfilePath}")
+        alpineImageId = hash.split(':')[1]
+      }
+    }
     if (params.scan_for_policy_violations) {
       stage('Evaluate Policies') {
```

Comment:
We have created a new IQ application called docker-nexus3-alpine where alpine image evaluations should be placed.